### DISCUSSION TITLE
Grouping by meaning or folder structure?

### DISCUSSION THREAD

**Discussion by NeilRiver** [payloadcms/payload#9756] [General]
Hello everyone! I've been studying the documentation for 3 days now and still haven't been able to spot any hints about grouping modules or folders.

I have a hosting sales project and the site has a lot of pages. And in my case (though I think for everyone) a folder structure of content would be useful.

Well, for example,

"hosting page"
-> our offers
-> we are the best because
-> our certificates

"page with s3"
-> tariffs
-> comparison with Amazon
-> configurations

and as far as I understand, all I can do is put all the modules in a row in 1 list?

**Comment by bratvanov** [comment]
Hi! You can use the core [Nested Docs plugin](https://payloadcms.com/docs/plugins/nested-docs) to add parent-child relationships to your collection documents.

While the plugin and list view currently don’t support visually indenting nesting levels, the list view includes a "Parent" column to indicate if a document is a child of another document.
